src/app/content/studyGuides/components/Filters.spec.tsx

```diff
@@ -49,4 +50,29 @@ describe('Filters', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('does not render ConnectedFilterList if user is logged out', () => {
+    store.dispatch(receiveLoggedOut());
+    const pageId = stripIdVersion(book.tree.contents[0].id);
+    store.dispatch(receiveStudyGuidesTotalCounts({
+      [pageId]: {
+        [HighlightColorEnum.Green]: 1,
+        [HighlightColorEnum.Yellow]: 1,
+      },
+    }));
+
+    const component = renderer.create(<Provider store={store}>
+      <MessageProvider>
+        <Filters />
+      </MessageProvider>
+    </Provider>);
+
+    let message: string | undefined;
+    try {
+      component.root.findByType(ConnectedFilterList);
+    } catch (e) {
+      message = e.message;
+    }
+    expect(message).toEqual('No instances found with node type: "Connect(FiltersList)"');
```

Comment:
This works, however, it may be a problem if a message will change in the future due to some dependencies upgrade
We've been using `expect(() => component.root.findByType(ConnectedFilterList)).toThrow();` approach in some of the other cases which seems fine also for this one

Reply:
this strategy with the try/catch comparing the message is necessary for async logic, we do that in other places. in this case i think piotr's suggestion would be better

Reply:
I tried that before, it throws the error anyways.

Reply:
@KarinaMendez2 Did you `expect` `() => component.root.findByType(ConnectedFilterList)` or only `component.root.findByType(ConnectedFilterList)` - the first will work, the second will throw an error